process/css-styleguide/input/index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Input</title>
+    <link rel="stylesheet" href="../heading/heading.css" />
+    <link rel="stylesheet" href="./input.css" />
+    <link rel="stylesheet" href="../spacing.css" />
+    <style>
+      .input-container--local {
+        padding: 1rem;
+      }
+    </style>
+  </head>
+  <body class="p-2">
+    <div class="input-container--local">
+      <input type="text" class="input mr-3" placeholder="Write something" />
+      <input
+        type="text"
+        class="input input--error"
+        placeholder="Write something"
+      />

```

Comment:
I don't get why you display twice the `error` variant: 
<img width="444" alt="Capture d’écran 2022-07-13 à 10 21 23" src="https://user-images.githubusercontent.com/30866152/178686329-4f175f97-f9fd-4737-b565-55859523b3a6.png">